src/migrations/migrate.rs

```diff
@@ -209,11 +209,16 @@ async fn _migrate(cli: &mut Connection, _options: &Options,
     };
 
     if let Some(db_migration) = &db_migration {
-        if !migrations.contains_key(&db_migration.name) {
-                let target_rev = target_rev.as_ref()
-                    .unwrap_or_else(|| &migrations.last().unwrap().0);
-                return fixup(cli, &ctx, &migrations,
-                             &db_migration, target_rev, migrate).await;
+        if let Some(last) = migrations.last() {
+            if !migrations.contains_key(&db_migration.name) {
+                    let target_rev = target_rev.as_ref()
+                        .unwrap_or_else(|| last.0);
+                    return fixup(cli, &ctx, &migrations,
+                                 &db_migration, target_rev, migrate).await;
+            }
+        } else {
+            anyhow::bail!("there are migrations in the database,
+                           but no migrations in the filesystem");
```

Comment:
```suggestion
                           but no migrations in the schema directory");
```

Reply:
Well, I have slight impression that "schema directory" can be thought of as anything. I.e. there is a "directory" in LDAP, and there are "directory services" in XMPP, so someone can assume and ask what we mean by "schema directory". Especially when it's project init on cloud, when they initialize their first project (perhaps after playing with some toy schema in the UI).

Note, this is not without a precedent: we use "filesystem" in `edgedb migration log --from-fs`, and we use the word in other error messages in migrations.

Reply:
I've put the specific directory name instead.